CMakeLists.txt

```diff
@@ -111,8 +111,6 @@ include(cmake/hpc-coding-conventions/cpp/cmake/3rdparty.cmake)
 if(NOT TARGET CLI11::CLI11)
   cpp_cc_git_submodule(cli11 BUILD PACKAGE CLI11 REQUIRED)
 endif()
-# For the moment do not try and use an external Eigen.
-cpp_cc_git_submodule(eigen)
```

Comment:
@matz-e : reference in `.gitmodules` needed?

Reply:
Moved down and will only get triggered if tests are requested - we still need Eigen to test the solvers.